src/null-safety/migration-guide.md

```diff
@@ -133,26 +133,11 @@ You can find contact details on the package page on [pub.dev][].
 Before migrating your package's code,
 update its dependencies to null-safe versions:
 
-1. Update `pubspec.yaml` to use null-safe releases
-   (as listed in the **Resolvable** column)
-   of its dependencies.
-   Omit `.x` suffixes to make version solving more flexible,
-   and _don't_ update the SDK minimum constraint.
-   For example, the `pubspec.yaml` file might look like this:
-   ```yaml
-   ...
-   environment:
-     sdk: '>=2.8.1 <3.0.0'
-
-   dependencies:
-     path: ^1.8.0-nullsafety
-     process: ^4.0.0-nullsafety
-
-   dev_dependencies:
-     pedantic: ^1.10.0-nullsafety
-   ```
+1. Run `dart pub upgrade --null-safety` to upgrade to the latest versions
```

Comment:
Do we want to tell folks to explicitly set *stable* versions as lower bounds?

Reply:
Done (under the `### Package version` section)